Running role on ubuntu 15.04/16.04 fails

I used the following vagrant box: https://github.com/kraksoft/vagrant-box-ubuntu/releases/download/15.04/ubuntu-15.04-amd64.box Ubuntu 15.04 (based on amd64 server iso file)

$ vagrant up
```
[...]
skipping: [default]
 _______________________________________________________
/ TASK: geerlingguy.mysql | Ensure MySQL is started and \
\ enabled on boot.                                      /
 -------------------------------------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||


failed: [default] => {"failed": true}
msg: Job for mysql.service failed. See "systemctl status mysql.service" and "journalctl -xe" for details.


FATAL: all hosts have already failed -- aborting
 ____________
< PLAY RECAP >
 ------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||


           to retry, use: --limit @/home/peteraba/site.retry

default                    : ok=18   changed=8    unreachable=0    failed=1   

Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
vagrant up  190.06s user 7.51s system 12% cpu 26:17.54 total

```
$ vagrant ssh
```
Welcome to Ubuntu 15.04 (GNU/Linux 3.19.0-15-generic x86_64)

 * Documentation:  https://help.ubuntu.com/
Last login: Sun Oct  4 09:38:18 2015 from 10.0.2.2

```
vagrant@vagrant-ubuntu-trusty:~$ systemctl status mysql.service
```
● mysql.service - MySQL Community Server
   Loaded: loaded (/lib/systemd/system/mysql.service; enabled; vendor preset: enabled)
   Active: activating (start-post) (Result: exit-code) since Sun 2015-10-04 09:48:19 GMT; 3min 22s ago
  Process: 10148 ExecStart=/usr/bin/mysqld_safe (code=exited, status=2)
  Process: 10145 ExecStartPre=/usr/share/mysql/mysql-systemd-start pre (code=exited, status=0/SUCCESS)
 Main PID: 10148 (code=exited, status=2);         : 10149 (mysql-systemd-s)
   CGroup: /system.slice/mysql.service
           └─control
             ├─10149 /bin/bash /usr/share/mysql/mysql-systemd-start post
             └─11170 sleep 1

Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: chmod: cannot access ‘/var/log/mysql.err’: No such file or directory
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: 151004 09:48:19 mysqld_safe Starting mysqld daemon with databases from /var/lib/mysql
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: /usr/bin/mysqld_safe: 126: /usr/bin/mysqld_safe: cannot create /var/log/mysql.err: Permission denied
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: /usr/bin/mysqld_safe: 1: eval: cannot create /var/log/mysql.err: Permission denied
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: touch: cannot touch ‘/var/log/mysql.err’: Permission denied
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: chown: cannot access ‘/var/log/mysql.err’: No such file or directory
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: chmod: cannot access ‘/var/log/mysql.err’: No such file or directory
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: 151004 09:48:19 mysqld_safe mysqld from pid file /var/run/mysqld/mysqld.pid ended
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: /usr/bin/mysqld_safe: 126: /usr/bin/mysqld_safe: cannot create /var/log/mysql.err: Permission denied
Oct 04 09:48:19 vagrant-ubuntu-trusty systemd[1]: mysql.service: main process exited, code=exited, status=2/INVALIDARGUMENT
```
vagrant@vagrant-ubuntu-trusty:~$ journalctl -xe
```
Oct 04 09:48:19 vagrant-ubuntu-trusty systemd[1]: Unit mysql.service entered failed state.
Oct 04 09:48:19 vagrant-ubuntu-trusty systemd[1]: mysql.service failed.
Oct 04 09:48:19 vagrant-ubuntu-trusty sudo[7954]: pam_unix(sudo:session): session closed for user root
Oct 04 09:48:19 vagrant-ubuntu-trusty sshd[634]: pam_unix(sshd:session): session closed for user vagrant
Oct 04 09:48:19 vagrant-ubuntu-trusty systemd[1]: mysql.service holdoff time over, scheduling restart.
Oct 04 09:48:19 vagrant-ubuntu-trusty systemd[1]: Starting MySQL Community Server...
-- Subject: Unit mysql.service has begun start-up
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
-- 
-- Unit mysql.service has begun starting up.
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: 151004 09:48:19 mysqld_safe Logging to '/var/log/mysql.err'.
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: touch: cannot touch ‘/var/log/mysql.err’: Permission denied
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: chmod: cannot access ‘/var/log/mysql.err’: No such file or directory
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: 151004 09:48:19 mysqld_safe Starting mysqld daemon with databases from /var/lib/mysql
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: /usr/bin/mysqld_safe: 126: /usr/bin/mysqld_safe: cannot create /var/log/mysql.err: Permission denied
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: /usr/bin/mysqld_safe: 1: eval: cannot create /var/log/mysql.err: Permission denied
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: touch: cannot touch ‘/var/log/mysql.err’: Permission denied
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: chown: cannot access ‘/var/log/mysql.err’: No such file or directory
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: chmod: cannot access ‘/var/log/mysql.err’: No such file or directory
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: 151004 09:48:19 mysqld_safe mysqld from pid file /var/run/mysqld/mysqld.pid ended
Oct 04 09:48:19 vagrant-ubuntu-trusty mysqld_safe[10148]: /usr/bin/mysqld_safe: 126: /usr/bin/mysqld_safe: cannot create /var/log/mysql.err: Permission denied
Oct 04 09:48:19 vagrant-ubuntu-trusty systemd[1]: mysql.service: main process exited, code=exited, status=2/INVALIDARGUMENT
Oct 04 09:49:19 vagrant-ubuntu-trusty sshd[4161]: Received disconnect from 10.0.2.2: 11: disconnected by user
Oct 04 09:49:19 vagrant-ubuntu-trusty sshd[4143]: pam_unix(sshd:session): session closed for user vagrant
Oct 04 09:49:19 vagrant-ubuntu-trusty systemd-logind[413]: Removed session 3.
-- Subject: Session 3 has been terminated
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
-- Documentation: http://www.freedesktop.org/wiki/Software/systemd/multiseat
-- 
-- A session with the ID 3 has been terminated.
Oct 04 09:51:37 vagrant-ubuntu-trusty sshd[11128]: Accepted publickey for vagrant from 10.0.2.2 port 57754 ssh2: RSA b2:76:da:59:08:26:0e:13:54:61:dc:48:9a:d2:71:ba
Oct 04 09:51:37 vagrant-ubuntu-trusty sshd[11128]: pam_unix(sshd:session): session opened for user vagrant by (uid=0)
Oct 04 09:51:37 vagrant-ubuntu-trusty systemd[1]: Started Session 4 of user vagrant.
-- Subject: Unit session-4.scope has finished start-up
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
-- 
-- Unit session-4.scope has finished starting up.
-- 
-- The start-up result is done.
Oct 04 09:51:37 vagrant-ubuntu-trusty systemd-logind[413]: New session 4 of user vagrant.
-- Subject: A new session 4 has been created for user vagrant
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
-- Documentation: http://www.freedesktop.org/wiki/Software/systemd/multiseat
-- 
-- A new session with the ID 4 has been created for the user vagrant.
-- 
-- The leading process of the session is 11128.
Oct 04 09:51:37 vagrant-ubuntu-trusty systemd[1]: Starting Session 4 of user vagrant.
-- Subject: Unit session-4.scope has begun start-up
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
-- 
-- Unit session-4.scope has begun starting up.
```
vagrant@vagrant-ubuntu-trusty:~$ cat /var/log/mysql/error.log 
```
2015-10-04 09:37:37 7335 [Warning] Using unique option prefix myisam-recover instead of myisam-recover-options is deprecated and will be removed in a future release. Please use the full name instead.
2015-10-04 09:37:37 7335 [Note] Plugin 'FEDERATED' is disabled.
2015-10-04 09:37:37 7335 [Note] InnoDB: Using atomics to ref count buffer pool pages
2015-10-04 09:37:37 7335 [Note] InnoDB: The InnoDB memory heap is disabled
2015-10-04 09:37:37 7335 [Note] InnoDB: Mutexes and rw_locks use GCC atomic builtins
2015-10-04 09:37:37 7335 [Note] InnoDB: Memory barrier is not used
2015-10-04 09:37:37 7335 [Note] InnoDB: Compressed tables use zlib 1.2.8
2015-10-04 09:37:37 7335 [Note] InnoDB: Using Linux native AIO
2015-10-04 09:37:37 7335 [Note] InnoDB: Not using CPU crc32 instructions
2015-10-04 09:37:37 7335 [Note] InnoDB: Initializing buffer pool, size = 128.0M
2015-10-04 09:37:37 7335 [Note] InnoDB: Completed initialization of buffer pool
2015-10-04 09:37:37 7335 [Note] InnoDB: The first specified data file ./ibdata1 did not exist: a new database to be created!
2015-10-04 09:37:37 7335 [Note] InnoDB: Setting file ./ibdata1 size to 12 MB
2015-10-04 09:37:37 7335 [Note] InnoDB: Database physically writes the file full: wait...
2015-10-04 09:37:37 7335 [Note] InnoDB: Setting log file ./ib_logfile101 size to 48 MB
2015-10-04 09:37:37 7335 [Note] InnoDB: Setting log file ./ib_logfile1 size to 48 MB
2015-10-04 09:37:37 7335 [Note] InnoDB: Renaming log file ./ib_logfile101 to ./ib_logfile0
2015-10-04 09:37:37 7335 [Warning] InnoDB: New log files created, LSN=45781
2015-10-04 09:37:37 7335 [Note] InnoDB: Doublewrite buffer not found: creating new
2015-10-04 09:37:38 7335 [Note] InnoDB: Doublewrite buffer created
2015-10-04 09:37:38 7335 [Note] InnoDB: 128 rollback segment(s) are active.
2015-10-04 09:37:38 7335 [Warning] InnoDB: Creating foreign key constraint system tables.
2015-10-04 09:37:39 7335 [Note] InnoDB: Foreign key constraint system tables created
2015-10-04 09:37:39 7335 [Note] InnoDB: Creating tablespace and datafile system tables.
2015-10-04 09:37:39 7335 [Note] InnoDB: Tablespace and datafile system tables created.
2015-10-04 09:37:39 7335 [Note] InnoDB: Waiting for purge to start
2015-10-04 09:37:39 7335 [Note] InnoDB: 5.6.25 started; log sequence number 0
2015-10-04 09:37:40 7335 [Note] Binlog end
2015-10-04 09:37:40 7335 [Note] InnoDB: FTS optimize thread exiting.
2015-10-04 09:37:40 7335 [Note] InnoDB: Starting shutdown...
2015-10-04 09:37:43 7335 [Note] InnoDB: Shutdown completed; log sequence number 1625977
2015-10-04 09:37:43 7372 [Warning] Using unique option prefix myisam-recover instead of myisam-recover-options is deprecated and will be removed in a future release. Please use the full name instead.
2015-10-04 09:37:43 7372 [Note] Plugin 'FEDERATED' is disabled.
2015-10-04 09:37:43 7372 [Note] InnoDB: Using atomics to ref count buffer pool pages
2015-10-04 09:37:43 7372 [Note] InnoDB: The InnoDB memory heap is disabled
2015-10-04 09:37:43 7372 [Note] InnoDB: Mutexes and rw_locks use GCC atomic builtins
2015-10-04 09:37:43 7372 [Note] InnoDB: Memory barrier is not used
2015-10-04 09:37:43 7372 [Note] InnoDB: Compressed tables use zlib 1.2.8
2015-10-04 09:37:43 7372 [Note] InnoDB: Using Linux native AIO
2015-10-04 09:37:43 7372 [Note] InnoDB: Not using CPU crc32 instructions
2015-10-04 09:37:43 7372 [Note] InnoDB: Initializing buffer pool, size = 128.0M
2015-10-04 09:37:43 7372 [Note] InnoDB: Completed initialization of buffer pool
2015-10-04 09:37:43 7372 [Note] InnoDB: Highest supported file format is Barracuda.
2015-10-04 09:37:43 7372 [Note] InnoDB: 128 rollback segment(s) are active.
2015-10-04 09:37:43 7372 [Note] InnoDB: Waiting for purge to start
2015-10-04 09:37:43 7372 [Note] InnoDB: 5.6.25 started; log sequence number 1625977
2015-10-04 09:37:43 7372 [Note] Binlog end
2015-10-04 09:37:43 7372 [Note] InnoDB: FTS optimize thread exiting.
2015-10-04 09:37:43 7372 [Note] InnoDB: Starting shutdown...
2015-10-04 09:37:45 7372 [Note] InnoDB: Shutdown completed; log sequence number 1625987
2015-10-04 09:37:45 7397 [Warning] Using unique option prefix myisam-recover instead of myisam-recover-options is deprecated and will be removed in a future release. Please use the full name instead.
2015-10-04 09:37:45 7397 [Note] Plugin 'FEDERATED' is disabled.
2015-10-04 09:37:45 7397 [Note] InnoDB: Using atomics to ref count buffer pool pages
2015-10-04 09:37:45 7397 [Note] InnoDB: The InnoDB memory heap is disabled
2015-10-04 09:37:45 7397 [Note] InnoDB: Mutexes and rw_locks use GCC atomic builtins
2015-10-04 09:37:45 7397 [Note] InnoDB: Memory barrier is not used
2015-10-04 09:37:45 7397 [Note] InnoDB: Compressed tables use zlib 1.2.8
2015-10-04 09:37:45 7397 [Note] InnoDB: Using Linux native AIO
2015-10-04 09:37:45 7397 [Note] InnoDB: Not using CPU crc32 instructions
2015-10-04 09:37:45 7397 [Note] InnoDB: Initializing buffer pool, size = 128.0M
2015-10-04 09:37:45 7397 [Note] InnoDB: Completed initialization of buffer pool
2015-10-04 09:37:45 7397 [Note] InnoDB: Highest supported file format is Barracuda.
2015-10-04 09:37:45 7397 [Note] InnoDB: 128 rollback segment(s) are active.
2015-10-04 09:37:45 7397 [Note] InnoDB: Waiting for purge to start
2015-10-04 09:37:45 7397 [Note] InnoDB: 5.6.25 started; log sequence number 1625987
2015-10-04 09:37:45 7397 [Note] Binlog end
2015-10-04 09:37:45 7397 [Note] InnoDB: FTS optimize thread exiting.
2015-10-04 09:37:45 7397 [Note] InnoDB: Starting shutdown...
2015-10-04 09:37:47 7397 [Note] InnoDB: Shutdown completed; log sequence number 1625997
151004 09:37:48 mysqld_safe Starting mysqld daemon with databases from /var/lib/mysql
2015-10-04 09:37:48 0 [Warning] TIMESTAMP with implicit DEFAULT value is deprecated. Please use --explicit_defaults_for_timestamp server option (see documentation for more details).
2015-10-04 09:37:48 0 [Note] /usr/sbin/mysqld (mysqld 5.6.25-0ubuntu0.15.04.1) starting as process 7845 ...
2015-10-04 09:37:48 7845 [Warning] Buffered warning: Changed limits: max_open_files: 1024 (requested 5000)

2015-10-04 09:37:48 7845 [Warning] Buffered warning: Changed limits: table_open_cache: 431 (requested 2000)

2015-10-04 09:37:48 7845 [Warning] Using unique option prefix myisam-recover instead of myisam-recover-options is deprecated and will be removed in a future release. Please use the full name instead.
2015-10-04 09:37:48 7845 [Note] Plugin 'FEDERATED' is disabled.
2015-10-04 09:37:48 7845 [ERROR] Function 'innodb' already exists
2015-10-04 09:37:48 7845 [Warning] Couldn't load plugin named 'innodb' with soname 'ha_innodb.so'.
2015-10-04 09:37:48 7845 [ERROR] Function 'federated' already exists
2015-10-04 09:37:48 7845 [Warning] Couldn't load plugin named 'federated' with soname 'ha_federated.so'.
2015-10-04 09:37:48 7845 [ERROR] Function 'blackhole' already exists
2015-10-04 09:37:48 7845 [Warning] Couldn't load plugin named 'blackhole' with soname 'ha_blackhole.so'.
2015-10-04 09:37:48 7845 [ERROR] Function 'archive' already exists
2015-10-04 09:37:48 7845 [Warning] Couldn't load plugin named 'archive' with soname 'ha_archive.so'.
2015-10-04 09:37:48 7845 [Note] InnoDB: Using atomics to ref count buffer pool pages
2015-10-04 09:37:48 7845 [Note] InnoDB: The InnoDB memory heap is disabled
2015-10-04 09:37:48 7845 [Note] InnoDB: Mutexes and rw_locks use GCC atomic builtins
2015-10-04 09:37:48 7845 [Note] InnoDB: Memory barrier is not used
2015-10-04 09:37:48 7845 [Note] InnoDB: Compressed tables use zlib 1.2.8
2015-10-04 09:37:48 7845 [Note] InnoDB: Using Linux native AIO
2015-10-04 09:37:48 7845 [Note] InnoDB: Not using CPU crc32 instructions
2015-10-04 09:37:48 7845 [Note] InnoDB: Initializing buffer pool, size = 128.0M
2015-10-04 09:37:48 7845 [Note] InnoDB: Completed initialization of buffer pool
2015-10-04 09:37:48 7845 [Note] InnoDB: Highest supported file format is Barracuda.
2015-10-04 09:37:48 7845 [Note] InnoDB: 128 rollback segment(s) are active.
2015-10-04 09:37:48 7845 [Note] InnoDB: Waiting for purge to start
2015-10-04 09:37:48 7845 [Note] InnoDB: 5.6.25 started; log sequence number 1625997
2015-10-04 09:37:48 7845 [Warning] No existing UUID has been found, so we assume that this is the first time that this server has been started. Generating a new UUID: 93249e5b-6a7b-11e5-83e2-0800274ac42f.
2015-10-04 09:37:48 7845 [Note] Server hostname (bind-address): '127.0.0.1'; port: 3306
2015-10-04 09:37:48 7845 [Note]   - '127.0.0.1' resolves to '127.0.0.1';
2015-10-04 09:37:48 7845 [Note] Server socket created on IP: '127.0.0.1'.
2015-10-04 09:37:48 7845 [Note] Event Scheduler: Loaded 0 events
2015-10-04 09:37:48 7845 [Note] /usr/sbin/mysqld: ready for connections.
Version: '5.6.25-0ubuntu0.15.04.1'  socket: '/var/run/mysqld/mysqld.sock'  port: 3306  (Ubuntu)
2015-10-04 09:38:16 7845 [Note] /usr/sbin/mysqld: Normal shutdown

2015-10-04 09:38:16 7845 [Note] Giving 0 client threads a chance to die gracefully
2015-10-04 09:38:16 7845 [Note] Event Scheduler: Purging the queue. 0 events
2015-10-04 09:38:16 7845 [Note] Shutting down slave threads
2015-10-04 09:38:16 7845 [Note] Forcefully disconnecting 0 remaining clients
2015-10-04 09:38:16 7845 [Note] Binlog end
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'partition'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_SYS_DATAFILES'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_SYS_TABLESPACES'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_SYS_FOREIGN_COLS'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_SYS_FOREIGN'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_SYS_FIELDS'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_SYS_COLUMNS'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_SYS_INDEXES'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_SYS_TABLESTATS'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_SYS_TABLES'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_FT_INDEX_TABLE'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_FT_INDEX_CACHE'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_FT_CONFIG'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_FT_BEING_DELETED'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_FT_DELETED'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_FT_DEFAULT_STOPWORD'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_METRICS'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_BUFFER_POOL_STATS'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_BUFFER_PAGE_LRU'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_BUFFER_PAGE'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_CMP_PER_INDEX_RESET'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_CMP_PER_INDEX'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_CMPMEM_RESET'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_CMPMEM'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_CMP_RESET'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_CMP'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_LOCK_WAITS'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_LOCKS'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'INNODB_TRX'
2015-10-04 09:38:16 7845 [Note] Shutting down plugin 'InnoDB'
2015-10-04 09:38:16 7845 [Note] InnoDB: FTS optimize thread exiting.
2015-10-04 09:38:16 7845 [Note] InnoDB: Starting shutdown...
2015-10-04 09:38:17 7845 [Note] InnoDB: Shutdown completed; log sequence number 1626007
2015-10-04 09:38:17 7845 [Note] Shutting down plugin 'PERFORMANCE_SCHEMA'
2015-10-04 09:38:17 7845 [Note] Shutting down plugin 'ARCHIVE'
2015-10-04 09:38:17 7845 [Note] Shutting down plugin 'BLACKHOLE'
2015-10-04 09:38:17 7845 [Note] Shutting down plugin 'MRG_MYISAM'
2015-10-04 09:38:17 7845 [Note] Shutting down plugin 'CSV'
2015-10-04 09:38:17 7845 [Note] Shutting down plugin 'MEMORY'
2015-10-04 09:38:17 7845 [Note] Shutting down plugin 'MyISAM'
2015-10-04 09:38:17 7845 [Note] Shutting down plugin 'sha256_password'
2015-10-04 09:38:17 7845 [Note] Shutting down plugin 'mysql_old_password'
2015-10-04 09:38:17 7845 [Note] Shutting down plugin 'mysql_native_password'
2015-10-04 09:38:17 7845 [Note] Shutting down plugin 'binlog'
2015-10-04 09:38:17 7845 [Note] /usr/sbin/mysqld: Shutdown complete

151004 09:38:17 mysqld_safe mysqld from pid file /var/run/mysqld/mysqld.pid ended
```

Note that it does seem to work with 14.04.

--- a/defaults/main.yml
+++ b/defaults/main.yml
@@ -48,14 +48,10 @@ mysql_max_connections: 151
 # Other settings.
 mysql_wait_timeout: 28800
 
-# Try number of CPU's * 2 for thread_concurrency.
-mysql_thread_concurrency: 2
-
 # InnoDB settings.
 # Set .._buffer_pool_size up to 80% of RAM but beware of setting too high.
 mysql_innodb_file_per_table: "1"
 mysql_innodb_buffer_pool_size: "256M"
-mysql_innodb_additional_mem_pool_size: "20M"
 # Set .._log_file_size to 25% of buffer pool size.
 mysql_innodb_log_file_size: "64M"
 mysql_innodb_log_buffer_size: "8M"

--- a/tasks/configure.yml
+++ b/tasks/configure.yml
@@ -42,6 +42,19 @@
     mode: 0644
   when: mysql_slow_query_log_enabled
 
+- name: Create error log file (if configured).
+  shell: "touch {{ mysql_log_error }} creates={{ mysql_log_error }}"
+  when: mysql_log == "" and mysql_log_error != ""
+
+- name: Set ownership on error log file (if configured).
+  file:
+    path: "{{ mysql_log_error }}"
+    state: file
+    owner: mysql
+    group: mysql
+    mode: 0644
+  when: mysql_slow_query_log_enabled
+
 - name: Ensure MySQL is started and enabled on boot.
   service: "name={{ mysql_daemon }} state=started enabled={{ mysql_enabled_on_startup }}"
   register: mysql_service_configuration

--- a/templates/my.cnf.j2
+++ b/templates/my.cnf.j2
@@ -78,13 +78,9 @@ max_connections = {{ mysql_max_connections }}
 # Other settings.
 wait_timeout = {{ mysql_wait_timeout }}
 
-# Try number of CPU's * 2 for thread_concurrency.
-thread_concurrency = {{ mysql_thread_concurrency }}
-
 # InnoDB settings.
 innodb_file_per_table = {{ mysql_innodb_file_per_table }}
 innodb_buffer_pool_size = {{ mysql_innodb_buffer_pool_size }}
-innodb_additional_mem_pool_size = {{ mysql_innodb_additional_mem_pool_size }}
 innodb_log_file_size = {{ mysql_innodb_log_file_size }}
 innodb_log_buffer_size = {{ mysql_innodb_log_buffer_size }}
 innodb_flush_log_at_trx_commit = {{ mysql_innodb_flush_log_at_trx_commit }}

